core/cmd/renderer.go

```diff
@@ -86,6 +91,50 @@ func (rt RendererTable) renderJobs(jobs []models.JobSpec) error {
 	return nil
 }
 
+func (rt RendererTable) renderConfiguration(cwl presenters.ConfigWhitelist) error {
+	table := rt.newTable([]string{"Key", "Value"})
+	
+	table.Append([]string{
+		"ACCOUNT_ADDRESS",
+		cwl.AccountAddress,
+	})
+
+	schemaT := reflect.TypeOf(orm.ConfigSchema{})
+	cwlT := reflect.TypeOf(cwl.Whitelist)
+	cwlV := reflect.ValueOf(cwl.Whitelist)
+
+	for index := 0; index < cwlT.NumField(); index++ {
+		item := cwlT.FieldByIndex([]int{index})
+		schemaItem, ok := schemaT.FieldByName(item.Name)
+		if !ok {
+			logger.Panicf("Field %s missing from store.Schema", item.Name)
+		}
+		envName, ok := schemaItem.Tag.Lookup("env")
+		if !ok {
+			continue
+		}
+		field := cwlV.FieldByIndex(item.Index)
+
+		if stringer, ok := field.Interface().(fmt.Stringer); ok {
+			if stringer != reflect.Zero(reflect.TypeOf(stringer)).Interface() {
+				table.Append([]string{
+					envName,
+					stringer.String(),
+				})
+			}
+		} else {
+			table.Append([]string{
+				envName,
+				fmt.Sprintf("%v", field),
+			})
```

Comment:
Was there a reason why using fmt.Sprintf("%s", ..) was not chosen instead?  It defaults to the Stringer interface when available.

Reply:
@felder-cl I'm not well versed in Go, but I didn't choose it for a specific reason. I was using the `(ConfigWhitelist).String` function in presenters.go as a template to get the same values as in the web interface. I'm happy to make a change / learn though, let me know.

Reply:
@felder-cl got any updates for me on this?

Reply:
Looks like its ok :+1: